src/annotator/config/index.js

```diff
@@ -76,75 +83,104 @@ function configurationKeys(appContext) {
  */
 const configDefinitions = {
   annotations: {
+    allowInBrowserExt: true,
+    defaultValue: null,
     getValue: settings => settings.annotations,
   },
   appType: {
-    getValue: settings =>
-      settings.hostPageSetting('appType', {
-        allowInBrowserExt: true,
-      }),
+    allowInBrowserExt: true,
+    defaultValue: null,
+    getValue: settings => settings.hostPageSetting('appType'),
   },
   branding: {
+    defaultValue: null,
+    allowInBrowserExt: false,
     getValue: settings => settings.hostPageSetting('branding'),
   },
   // URL of the client's boot script. Used when injecting the client into
   // child iframes.
   clientUrl: {
+    allowInBrowserExt: true,
+    defaultValue: null,
     getValue: settings => settings.clientUrl,
   },
   enableExperimentalNewNoteButton: {
+    allowInBrowserExt: false,
+    defaultValue: null,
```

Comment:
Since this is a boolean, I think it would make sense for the default value to be `false`.